### PR TITLE
Improvement/paypal 6 upgrade document

### PR DESCRIPTION
**Pull request by mttdffy**

### CODE REVIEW

**Review by jyucis — APPROVED** [review]

**Review by ebigler — COMMENTED** [review]
This is great. Do you think we should also include any guidance around making use of unbranded credit cards and how that functionality is congifured and styled? I know we said this probably wouldn't make sense for a client who's upgrading, but I don't think we mention it at all in the readme (it could be useful for a new client).

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Upgrading from v2.x to v3.0.0
```
Comment (bencrouse) [inline UPGRADE.md]:
```suggestion
Upgrading This Gem from v2.x to v3.0.0
```

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+================================================================================
+
+As of v3.0, this plugin utilizes the [PayPal Commerce Platform](https://www.paypal.com/us/webapps/mpp/commerce-platform). By default this integrates PayPal's [Smart Payment Buttons](https://developer.paypal.com/docs/commerce-platform/payment/checkout/) into your Workarea Application to allow customers to use any payment method supported in their area to complete checkout on your storefront.
```
Comment (bencrouse) [inline UPGRADE.md]:
```suggestion
As of this gem's v3.0 version, this plugin utilizes the [PayPal Commerce Platform](https://www.paypal.com/us/webapps/mpp/commerce-platform). By default this integrates PayPal's [Smart Payment Buttons](https://developer.paypal.com/docs/commerce-platform/payment/checkout/) into your Workarea Application to allow customers to use any payment method supported in their area to complete checkout on your storefront.
```

**Review by bencrouse — APPROVED** [review]

**Comment by bencrouse** [comment]
@ebigler there's a section in the README on that.

**Pull request merged**